<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistCreation</class>
 <widget class="QDialog" name="PlaylistCreation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Study Playlist Creation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="title">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Study Playlist Creation</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="difficulty">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="difficulty_appeal">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Choose the level of difficulty:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="difficulty_choice">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
padding-left: 8px;
}</string>
           </property>
           <item>
            <property name="text">
             <string>Random</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Easy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hard</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="study_type">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="study_type_appeal">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Choose the study type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="study_type_choice">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
padding-left: 8px;
}</string>
           </property>
           <item>
            <property name="text">
             <string>Random</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Revision</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Learning</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="flip_mode_button">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Start studying in Flip Mode</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover {
background-color: rgba(165, 165, 165, 30)
}</string>
          </property>
          <property name="text">
           <string>Flip Mode</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../assets/flip.png</normaloff>../../assets/flip.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quiz_mode_button">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Start studying in Quiz Mode</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:hover {
background-color: rgba(165, 165, 165, 30)
}</string>
          </property>
          <property name="text">
           <string>Quiz Mode</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../assets/quiz.png</normaloff>../../assets/quiz.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
